pandas/tseries/tests/test_resample.py

```diff
@@ -769,6 +769,36 @@ def test_resample_empty_dtypes(self):
                         # (ex: doing mean with dtype of np.object)
                         pass
 
+    def test_resample_loffset_arg_type(self):
+        # GH 13218, 15002
+        df = self.create_series().to_frame('value')
+        expected_means = [df.values[i:i + 2].mean()
+                          for i in range(0, len(df.values), 2)]
+        expected_index = self.create_index(df.index[0],
+                                           periods=len(df.index) / 2,
+                                           freq='2D')
+        # loffset coreces PeriodIndex to DateTimeIndex
```

Comment:
`In [19]: df = pd.DataFrame(data=list(range(10)), index=pd.period_range("1/1/2000", freq="D", periods=10))

In [20]: df.resample("2D").mean().index
Out[20]:  DatetimeIndex(['2000-01-01', '2000-01-03', '2000-01-05', '2000-01-07',
               '2000-01-09'],
              dtype='datetime64[ns]', freq='2D')
`